plugins/org.jkiss.dbeaver.ext.generic/plugin.xml

```diff
@@ -876,14 +876,31 @@
                     class="com.facebook.presto.jdbc.PrestoDriver"
                     sampleURL="jdbc:presto://{host}:{port}/{database}"
                     defaultPort="8080"
-                    description="PrestoDB JDBC driver"
+                    description="Facebook Presto (prestodb) JDBC driver"
                     webURL="https://prestodb.io/"
                     categories="sql,analytic,bigdata">
                     <file type="jar" path="maven:/com.facebook.presto:presto-jdbc:RELEASE"/>
 
                     <parameter name="use-search-string-escape" value="true"/>
                 </driver>
 
+                <!-- PrestoSQL -->
+                <driver
+                        id="prestosql_jdbc"
+                        label="PrestoSQL"
+                        icon="icons/presto_icon.png"
+                        iconBig="icons/presto_icon_big.png"
+                        class="io.prestosql.jdbc.PrestoDriver"
+                        sampleURL="jdbc:presto://{host}:{port}/{database}"
+                        defaultPort="8080"
+                        description="Community Presto (prestosql) JDBC driver"
```

Comment:
```suggestion
                        description="Presto (prestosql) JDBC driver"
```

Reply:
Agree. Community in this context may be misleading